frontend/src/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.svg';
```

Comment:
why do you need this?

Reply:
it's related to the webpack installing. it was for adding svg images